Makefile

```diff
@@ -14,7 +14,7 @@ GO_MOD_VERSION=$(shell hack/go-mod-version.sh)
 
 all: check build
 
-check: fmt check-uncommitted lint test/unit
+check: lint test/unit
```

Comment:
They `fmt` and `check-uncommitted` were not removed.
The commit message is unclear.